docker-hub/scim.md

```diff
@@ -2,175 +2,63 @@
-
-You must run a full-sync after enabling SCIM, if you already have users assigned to the Docker Hub app. This provisions the users that are assigned in the IdP Directory to Docker Hub.
-
-1. Navigate to **Applications** > **Applications** and select the Docker Hub app.
-2. In the **Assignments** tab, select **Provision User** if you have pending users.
-3. Click **Apply to All** > **Reapply Mappings** and **Confirm**.
-
-    > **Note**
-    >
-    > This will generate a full-sync and any user that wasn't previously provisioned is now provisioned in Docker Hub.
-
-![provision-user](images/provision-user.png){:width="700px"}
-
-### Azure AD
-
-1. Navigate to Azure AD and select **AzureAD admin**.
-2. In the **Default Directory**, select **Add** > **Enterprise Application** > **Create your own application**.
-
-    > **Note**
-    >
-    > When you create an Enterprise Application you can configure SCIM with
-    > Security Assertion Markup Language (SAML).
-
-3. Type **Docker** for application name, select **non-gallery**, and **Create**.
-4. In your Docker application, navigate to **Provisioning**, and select **Get Started**.
-5. Select **Automatic** for the provisioning mode and enter your SCIM credentials.
-
-    > **Note**
-    >
-    > You can access and copy your SCIM URL (Tenant URL) and API Token (Secret
-    > Token) in Docker Hub.
-
-6. Select **Test Connection** to enable and authorize the provisioning.
-7. In **Mappings**, select **Provision Azure Active Directory Groups** to disable Groups and **Save**.
-
-    > **Note**
-    >
-    > You must turn off group sync to avoid having your configuration
-    > quarantined.
-8. Select **Provision Azure Active Directory Users**, in **Attribute Mappings**, and keep the **userName**, **Active**, **givenName**, **familyName**. Delete the other attributes listed.
-
-    ![attribute-mapping](images/attribute-mapping.png){:width="700px"}
-
-9. Select **Start Provisioning** to begin the full synchronization.
+## Set up SCIM
 
-     > **Note**
-     >
-     > It can take up to 30 minutes to begin provisioning. You can also
-     > provision on demand provisioning with one user that's already assigned
-     > to the application.
+You must make sure you have [configured SSO](../single-sign-on/index.md) before you enable SCIM. Enforcing SSO is not required.
 
-### Onelogin
+### Step one: Enable SCIM in Docker Hub
 
-1. In Onelogin, navigate to **Applications** > **Applications** > **Add app**.
-2. In the search field, enter **SCIM Provisioner with SAML (SCIM v2 Core)** and select the item in the results.
-3. Enter **Docker Hub** as the display name and **Save**.
-4. Navigate to the left navigation, and select **Configuration**.
-5. In a separate tab, navigate to **Docker Hub** > **Settings** > **Security** > **SSO** > **SAML**.
-6. Copy the following fields from Docker Hub, in to OneLogin:
+1. In the **Single Sign-On Connections** table, select the **Actions** icon and **Setup SCIM**.
```

Comment:
I made a judgment call here. I feel like since the admins will already be familiar with the settings area, they will be able to navigate there fine so adding the info seems repetitive and redundant. After this goes out, I want to create a follow-up PR and streamline the 'manage' page further